bin/morgue.js

```diff
@@ -6343,10 +6343,8 @@ function coronerDelete(argv, config) {
   }
 
   if (aq && aq.query) {
-    coroner.promise('query', p.universe, p.project, aq.query).then(function(r) {
-      unpackQueryObjects(o, r);
-      return delete_fn();
-    }).then(std_success_cb).catch(std_failure_cb);
+    coroner.promise('delete_query', p.universe, p.project, aq.query,
```

Comment:
Since this basically just submits a delete by query to coronerd without any knowledge of how things went, perhaps we should at least let the user know that we have done that.  The change here removes morgue's dialogue about how things are going, so folks might be confused.

Do we want to keep the possibility of using the query & batched delete logic in case delete by query drops coronerd?